static_topology/stackstate_checks/static_topology/static_topology.py

```diff
@@ -0,0 +1,136 @@
+# (C) StackState 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import csv
+import codecs
+
+from stackstate_checks.base import ConfigurationError, AgentCheck, TopologyInstance
+
+
+class StaticTopologyCheck(AgentCheck):
+    SERVICE_CHECK_NAME = "StaticTopology"
+    INSTANCE_TYPE = "StaticTopology"
+
+    def get_instance_key(self, instance):
+        if "components_file" not in instance:
+            raise ConfigurationError('Static topology instance missing "components_file" value.')
+
+        return TopologyInstance(self.INSTANCE_TYPE, instance["components_file"])
+
+    def check(self, instance):
+        if 'components_file' not in instance:
+            raise ConfigurationError('Static topology instance missing "components_file" value.')
+        if 'relations_file' not in instance:
+            raise ConfigurationError('Static topology instance missing "relations_file" value.')
+        if 'type' not in instance:
+            raise ConfigurationError('Static topology instance missing "type" value.')
```

Comment:
we don't throw exception from the check but rather use the `service_check` critical

Reply:
@TizianoPerrucci We don't create `service_checks` for config params missing. As an example:- https://github.com/StackVista/stackstate-agent-integrations/blob/04230dabb5062f23a643a46d644d71172c5479b4/sap/stackstate_checks/sap/sap.py#L37